sklearn/utils/_mocking.py

```diff
@@ -147,9 +147,13 @@ def _check_X_y(self, X, y=None, should_be_fitted=True):
         Parameters
         ----------
         X : array-like of shape (n_samples, n_features)
-            The data set.
+            The data set. Required argument.
```

Comment:
```suggestion
            The data set.
```

Reply:
Adressed in https://github.com/scikit-learn/scikit-learn/pull/25997/commits/bf778271bac9af82a1446872ece443108fb2cb18.